### ISSUE TITLE
aio module not included in 0.2.0 release

### ISSUE THREAD

**Issue by mackeyja92** [Peter-Slump/python-keycloak-client#11]
When installing the package from PyPI the `aio`module is not included.

```bash
$ pip3 install python-keycloak-client[aio]
Collecting python-keycloak-client[aio]
  ... truncated
Successfully built python-keycloak-client future yarl
Installing collected packages: certifi, chardet, urllib3, idna, requests, pyasn1, rsa, ecdsa, future, python-jose, multidict, yarl, attrs, async-timeout, aiohttp, python-keycloak-client
Successfully installed aiohttp-3.4.4 async-timeout-3.0.1 attrs-18.2.0 certifi-2018.11.29 chardet-3.0.4 ecdsa-0.13 future-0.17.1 idna-2.7 multidict-4.5.2 pyasn1-0.4.4 python-jose-3.0.1 python-keycloak-client-0.2.0 requests-2.20.1 rsa-4.0 urllib3-1.24.1 yarl-1.2.6

$ python -c "import keycloak; from keycloak.aio.realm import KeycloakRealm"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'keycloak.aio'

$ cd "$(python -c "import sys; print(sys.path[-1]);")/keycloak";
$ pwd
~/.venvs/test/lib/python3.7/site-packages/keycloak

$ ls -lah
total 96
drwxr-xr-x  13 user  group   416B Dec  8 10:01 .
drwxr-xr-x  60 user  group   1.9K Dec  8 10:01 ..
-rw-r--r--   1 user  group     0B Dec  8 10:01 __init__.py
drwxr-xr-x  11 user  group   352B Dec  8 10:01 __pycache__
drwxr-xr-x   8 user  group   256B Dec  8 10:01 admin
-rw-r--r--   1 user  group   5.5K Dec  8 10:01 authz.py
-rw-r--r--   1 user  group   3.0K Dec  8 10:01 client.py
-rw-r--r--   1 user  group   251B Dec  8 10:01 exceptions.py
-rw-r--r--   1 user  group   554B Dec  8 10:01 mixins.py
-rw-r--r--   1 user  group    12K Dec  8 10:01 openid_connect.py
-rw-r--r--   1 user  group   2.2K Dec  8 10:01 realm.py
-rw-r--r--   1 user  group   7.2K Dec  8 10:01 uma.py
-rw-r--r--   1 user  group   962B Dec  8 10:01 well_known.py
```

**Comment by mackeyja92** [comment]
@Peter-Slump will you be publishing a new version to fix the issue in PyPI as well?

Thanks!

**Comment by Peter-Slump** [comment]
@mackeyja92 you where too quick ;) I've just released a new version (0.2.1) to PyPI including aio. If you install the library again you will find the aio implementation of @nkoshell 
Sorry for the inconvenience.
